executable/AndroidManifest.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: android_manifest
+  title: Android Binary manifest xml
+  file-extension: xml
```

Comment:
Is it *really* ".xml" file, and in reality, it's not really XML, it's some kind of binary format? If so, we also need to note it specifically in top-level `doc`.

Reply:
It is a binary format and the extension is really .xml. It is in fact an XML file that has been translated into a binary file. It has a lot of the structure of XML, just not any of the XML markup.